src/components/Species/index.tsx

```diff
@@ -750,12 +805,40 @@ export default function SpeciesList({ reloadData, species }: SpeciesListProps):
               iconRight='cancel'
               onClickRightIcon={clearSearch}
             />
-            <SpeciesFilters filters={record} setFilters={setRecord} />
+            <IconButton onClick={handleFilterClick} size='small' className={classes.iconContainer}>
+              <Icon name='filter' size='medium' className={classes.icon} />
+            </IconButton>
+            <Popover
+              id='simple-popover'
+              open={Boolean(filterAnchorEl)}
+              anchorEl={filterAnchorEl}
+              onClose={handleFilterClose}
+              anchorOrigin={{
+                vertical: 'bottom',
+                horizontal: 'center',
+              }}
+              transformOrigin={{
+                vertical: 'top',
+                horizontal: 'center',
+              }}
+              className={classes.popoverContainer}
+            >
+              <FilterGroup
+                initialFilters={filters}
+                fields={filterColumns}
```

Comment:
thoughts on hiding (or marking the dropdowns as disabled) if there are no values to select from? maybe not an issue in production.. in staging we have orgs with dummy species.. a lot of the filter dropdowns have no values to select from... ofc the data is mostly test data.

Reply:
That would make sense. I'll look at doing that